internal/ceb/ssh.go

```diff
@@ -0,0 +1,263 @@
+package ceb
+
+import (
+	"context"
+	"crypto/x509"
+	"encoding/base64"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"strconv"
+	"syscall"
+
+	"github.com/creack/pty"
+	"github.com/gliderlabs/ssh"
+	"github.com/hashicorp/go-hclog"
+	"github.com/pkg/errors"
+	gossh "golang.org/x/crypto/ssh"
+)
+
+func RunExecSSHServer(ctx context.Context, logger hclog.Logger, sport, hostkey, key string) error {
```

Comment:
Given the complexity of this (its a full SSH server), I'd love to see some unit tests on this. I think it should be easy enough to spin up one in-memory and use an SSH client to test that it works on *nix-like machines (macOS, Linux).